src/lachesis/lahcesis_test.go

```diff
@@ -86,7 +86,7 @@ func runNode(t testing.TB, logger *logrus.Logger, config *node.Config,
 	db := poset.NewInmemStore(participants, config.CacheSize, nil)
 	app := dummy.NewInmemDummyApp(logger)
 	selectorArgs := node.SmartPeerSelectorCreationFnArgs{
-		LocalAddr: localAddr,
+		LocalAddr:    localAddr,
```

Comment:
Ensure you run `gofmt -s -w` before submitting PRs

Reply:
OK, I will pass that to prev commit author.